data/pathfinder/paizo/player_companion/weapon_masters_handbook/wmh_abilities.lst

```diff
@@ -60,7 +60,7 @@ Weapon Specialization			KEY:Tempered Champion ~ Weapon Specialization
+Varisian Free-Style Fighter		KEY:Fighter Archetype ~ Varisian Free-Style Fighter								NAMEISPI:YES	CATEGORY:Archetype		TYPE:Archetype.FighterArchetype																									PRECLASS:1,Paladin=1		PREMULT:1,[PREABILITY:1,CATEGORY=Archetype,Paladin Archetype ~ Varisian Free-Style Fighter],[!PREFACT:1,ABILITIES,Fighter_Archetype_ClassSkills=True,Fighter_Archetype_BonusFeat1=True,Fighter_Archetype_BonusFeat6=True,Fighter_Archetype_BonusFeat10=True,Fighter_Archetype_BonusFeat12=True,Fighter_Archetype_WeaponTraining=True,Fighter_Archetype_WeaponMastery=True,Fighter_Archetype_ArmorTraining=True]																															DESCISPI:YES	DESC:Free-style fighters are most common in Varisia, where their training-students' traveling study with multiple teachers, or perhaps instruction in the anything-goes mentality of the Price of Freedom school in Kaer Maga-allows them to blend fighting styles into a greater whole.																																																																																																																																																																																																																													ABILITY:Internal|AUTOMATIC|Varisian Free-Style Fighter ~ Class Skills|PRECLASS:1,Fighter=1	ABILITY:Special Ability|AUTOMATIC|Varisian Free-Style Fighter ~ Martial Flexibility|PRECLASS:1,Fighter=1	ABILITY:Special Ability|AUTOMATIC|Varisian Free-Style Fighter ~ Free Fighting Style|PRECLASS:1,Fighter=3																																																																																																																																																																																																																																																																																																COST:0	SOURCEPAGE:p.7																											FACT:Fighter_Archetype_ClassSkills|true	FACT:Fighter_Archetype_BonusFeat1|true	FACT:Fighter_Archetype_BonusFeat6|true	FACT:Fighter_Archetype_BonusFeat10|true	FACT:Fighter_Archetype_BonusFeat12|true	FACT:Fighter_Archetype_WeaponTraining|true	FACT:Fighter_Archetype_WeaponMastery|true	FACT:Fighter_Archetype_ArmorTraining|true	FACT:Fighter_CF_ClassSkills|true	FACT:Fighter_CF_BonusFeat1|true	FACT:Fighter_CF_BonusFeat6|true	FACT:Fighter_CF_BonusFeat10|true	FACT:Fighter_CF_BonusFeat12|true	FACT:Fighter_CF_WeaponTraining|true	FACT:Fighter_CF_WeaponMastery|true	FACT:Fighter_CF_ArmorTraining|true
```

Comment:
Uh, this makes no sense `Fighter Archetype ~ Varisian Free-Style Fighter` and then `PRECLASS:1,Paladin=1` and then `Paladin Archetype ~ Varisian Free-Style Fighter`  This should all be Fighter.

Reply:
You're right, should be fixed now.